src/solver/expressions/include/antares/solver/expressions/visitors/NodeVisitor.h

```diff
@@ -269,5 +293,13 @@ class NodeVisitor: public IName
      * @return The result of processing the ComponentParameterNode.
      */
     virtual R visit(const Nodes::ComponentParameterNode*, Args... args) = 0;
+
+    void setNodeVisitorCustomLog(const NodeVisitorCustomLog& nodeVisitorCustomLog)
+    {
+        nodeVisitorCustomLog_ = nodeVisitorCustomLog;
+    }
```

Comment:
About : **setNodeVisitorCustomLog(...)**
is unused. 
Removal ?

Reply:
The client of this class has to use this method in order to customize logging;

For instance the client would create an  NodeVisitorCustomLog cust_log = {.info = my_info_log, .warning =etc...};
where my_info could be:

void my_info(const std::string& msg){
logs.info(msg);
}

Reply:
[update] the default log is now yuni::logs, i can safely remove the setter